component/src/main/java/com/celements/photo/container/ImageUrlDim.java

```diff
@@ -0,0 +1,42 @@
+package com.celements.photo.container;
+
+import com.google.common.base.Optional;
+
+import groovy.lang.Immutable;
+
+@Immutable
+public class ImageUrlDim {
```

Comment:
why not making it a smart class with static factory methods breaking up the url String into the base url and the width and height dimensions returning a ImageUrlDim object.
Alternative you could use the builder pattern. The builder would then do the parse logic in a setURL, but it would to support to use the builder with setURL and overwrite or add width and hight with setWidth and setHeight or overwrite the ...